core/shared/src/main/scala/scalaz/zio/Task.scala

```diff
@@ -0,0 +1,631 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+/*
+ * PHASE 1 DONE
+ * PHASE 2 DONE
+ * PHASE 3 DONE
+ * PHASE 4 DONE
+ * PHASE 6 DONE
+ */
+
+/*
+ * EXTENDS ZIO_E_Throwable
+ */
+object Task {
+
+  def apply[A](a: => A): Task[A] = effect(a)
+
+  type TaskE = Throwable
+  type TaskR = Any
```

Comment:
I'd delete:

1. All the temp comments
2. The type aliases for `TaskE` and `TaskR`
3. Scaladoc for all the functions (instead, we can link to ZIO sScaladoc)